src/utils/helpers.ts

```diff
@@ -51,3 +51,21 @@ export const buildListsUrl = (url: string, request?: ListsRequest) => {
 
   return url;
 };
+
+export const removeSpaces = (object: {[key: string]: boolean | string}) => {
+  const result = {} as {[key: string]: boolean | string};
+
+  for (const key in object) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (object.hasOwnProperty(key)) {
+      const value = object[key];
+      if (typeof value === 'string') {
+        result[key] = value.trim();
+      } else {
+        result[key] = value;
+      }
+    }
+  }
+
+  return result;
+};
```

Comment:
This function feels like it would make more sense with a looser type for `object`: it only actually does anything special with `string`-valued properties and skips anything that isn't a `string`, so there's no reason to restrict it to just strings and booleans. Normally, I'm all for tightening up types as much as possible, but given the semantics of this function, where it only affects string properties and ignores others, I lean toward loosening up types here